packages/admin/cms-admin/src/dam/DataGrid/FolderDataGrid.tsx

```diff
@@ -243,6 +246,15 @@ const FolderDataGrid = ({
         setHoveredId(null);
     };
 
+    const getSnackbarElement = (message: string) => (
```

Comment:
Why did you create this function? It is only used in one place...

Reply:
I had more different messages in the beginning but then realized I don't need them. This is a relict so I can remove it if you want :)